static/markdown/a-confluence-of-waters.md

```diff
@@ -0,0 +1,76 @@
+![](https://two.compost.digital/images/title-images/bluemarbles-700x350.jpg)
+
+# A Confluence of Waters
```

Comment:
We need to manually add each article's markdown version, as HTML and frontmatter is not supported. Basically a basic version of each.